services-js/registry-certs/pages/_document.tsx

```diff
@@ -7,7 +7,7 @@ import { extractCritical } from 'emotion-server';
 import { StatusModal, ContactForm } from '@cityofboston/react-fleet';
```

Comment:
Since you removed `emotion-server` from the `package.json` you’ll need to remove it from this file as well, and you can remove the `__NEXT_DATA__` shenanigans from here and `_app.tsx`

Reply:
thanks for the catch!